src/main/java/fr/moribus/imageonmap/ui/MapItemManager.java

```diff
@@ -239,9 +240,10 @@ static private void onItemFrameRemove(ItemFrame frame, Player player, EntityDama
         
     }
     
-    @EventHandler
+    @EventHandler(priority = EventPriority.HIGH)
```

Comment:
This should be `EventPriority.HIGHEST`, as the event needs the final cancellation state from other plugins (but cannot watch on `MONITOR` priority because we cancels the event internally).

We'll fix that after merge, this is only an advice.